src/main/java/com/monumental/util/application/InitialDataLoadApplication.java

```diff
@@ -19,16 +27,23 @@
  * Then does validation on each CsvMonumentConverterResult's Monument before inserting it into the database
  * Finally, closes the CsvFileReader
  */
+@Configuration
+@SpringBootApplication
+@ComponentScan("com.monumental")
```

Comment:
What does this do?

Reply:
component scan tells it to discover autowireable classes in that package. By default it scans the package that the application is in which would mean this one wouldn't see any of the services etc.